docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+
+#### Helper functions ####
+
+usage()
+{
+    echo "Usage: $0 -n <NAMESPACE> [-d <DAYS_OF_LOGS>]"
+    echo "When using the -d flag, <DAYS_OF_LOGS> must be between 1 and 7"
+    echo "Example: '$0 -n codacy -d 3' would retrieve the last 3 days of logs from the 'codacy' namespace of the kubernetes cluster"
+    exit 3
+}
+
+date_days_ago() {
+    OS=$(uname)
+    if [[ "$OS" == "Darwin" ]]; then
+        date -v -$1\d "+%Y-%m-%d"
+    elif [[ "$OS" == "Linux" ]]; then
+        date -d '-$1 day' "+%Y-%m-%d"
+    else
+       echo "Unsupported operating system '$OS'"
+       exit 11
+    fi
+}
+
+cleanup()
+{
+    if [ -d "$LOGS_DIR" ]; then
+        rm -r $LOGS_DIR &>/dev/null
+    fi
+}
+trap cleanup EXIT
+
+#### Log extraction script ####
+
+while getopts "n:d:" option; do
+    case ${option} in
+        n )
+            NAMESPACE=${OPTARG}
+            ;;
+        d )
+            if [ $OPTARG -le 0 ] || [ $OPTARG -ge 8 ]; then
+                usage
+            else
+                DAYS=$((${OPTARG}-1)) # Day 0 is today
+            fi
+            ;;
+        \? )
+            usage
+            ;;
+    esac
+done
+
+[ -z "$NAMESPACE" ] && usage
+
+CURRENT_DATE_TIME=$(date "+%Y%m%d-%H%M%S")
+LOGS_DIR="$(mktemp -d)/codacy_logs_$CURRENT_DATE_TIME"
+
+echo "Starting log files extraction"
+
+# Check if zip is available
+echo "Checking if zip is installed..."
+if ! zip --version &>/dev/null; then
+    echo "zip not installed"
+    echo "Please install zip"
+    echo "Log files extraction failed, exiting..."
+    exit 4
+fi
+
+
+# Check if kubectl or microk8s.kubectl is available
+echo "Checking if kubectl is installed..."
+KUBECTL=$(which kubectl || which microk8s.kubectl)
+if [ $? -ne 0 ]; then
+    echo "kubectl not installed"
+    echo "Please install kubectl version specified in Codacy's documentation (see here - https://codacy.github.io/chart/#preparing-to-install-codacy) or the version used when installing your cluster"
```

Comment:
This section doesn't exist anymore https://codacy.github.io/chart/#preparing-to-install-codacy . There reference to kubectl was moved to this page https://codacy.github.io/chart/install/